src/main/java/org/kainos/ea/DropwizardTheITCrowdServiceConfiguration.java

```diff
@@ -14,7 +14,7 @@ public class DropwizardTheITCrowdServiceConfiguration extends Configuration {
 
   @JsonProperty("swagger")
   public SwaggerBundleConfiguration getSwagger() {
-    swagger.setResourcePackage("org.kainos.ea.controller");
+    swagger.setResourcePackage("org.kainos.ea.resources");
```

Comment:
Should this have been changed?

Reply:
Yip, was changed in an early commit. Team decided to keep resources.